src/api/index.ts

```diff
@@ -1,11 +1,20 @@
 import { QueryClient } from 'react-query';
 import axios, { AxiosInstance } from 'axios';
 
-export const queryClient = new QueryClient();
+export const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+      refetchOnMount: false,
+      refetchOnWindowFocus: false,
+      refetchOnReconnect: false,
```

Comment:
I don't think this setting should be global, it's better to make base config (and extend it if needed) and pass it into our custom useQuery hooks.